### ISSUE TITLE
Create testsuite for S2E

### ISSUE THREAD

**Issue by vitalych** [S2E/s2e-env#249]
Can be used both for checking PR correctness and benchmark performance.
The testsuite should use s2e-env to create test projects from pre-defined binaries.
Each project can then be run in parallel.

The first part is to write the testsuite
The second part is to integrate it in Jenkins, which is a separate issue.

**Comment by vitalych** [comment]
Also add:
- Tutorial with echo
- Export project at the end of each test run
